org.freedownloadmanager.Manager.yaml

```diff
@@ -83,8 +91,8 @@ modules:
         path: icons/org.freedownloadmanager.Manager-256.png
 
       - type: extra-data
-        url: https://dn3.freedownloadmanager.org/6/latest/freedownloadmanager.deb
-        sha256: 164a11bc5ff634e74f902c6334d159f608cdf45c037a0a3497dd1236bbd346c5
+        url: https://files2.freedownloadmanager.org/6/latest/freedownloadmanager.deb
```

Comment:
The unversioned URL here is essentially asking for trouble. If the size changes, it will fail to install for users.

Reply:
And it changed already...